.github/workflows/test_and_deploy.yml

```diff
@@ -32,12 +32,26 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: 3.8
+
+        # Temporary fix for 'pip install imageio-ffmpeg'
+        # not including the FFMPEG binary on Apple Silicon macs
+        # This step can be removed when issue is fixed in imageio-ffmpeg
+        # https://github.com/imageio/imageio-ffmpeg/issues/71
+      - name: brew install ffmpeg
+        if: runner.os == 'macOS'
+        uses: artemnovichkov/action-homebrew@0.1.0
```

Comment:
Are you sure that this action is needed? 
As I remember, the brew is preinstalled. In this action, brew is used without installation https://github.com/4DNucleome/PartSegCore-compiled-backend/blob/master/.github/workflows/wheels.yml